public/app/features/explore/LogsContainer.tsx

```diff
@@ -87,6 +87,11 @@ export class LogsContainer extends PureComponent<LogsContainerProps> {
     return [];
   };
 
+  deduplicateLogRows = (rows: LogRowModel[]) => {
```

Comment:
I wonder if it would make sense to have this next to here https://github.com/grafana/grafana/blob/d9e9843a106d5056074ed883eac9baa355913abc/public/app/features/explore/utils/ResultProcessor.ts#L107. One thing that would be solved is that right now we pass:
```
logRows={deduplicatedLogRows}
dedupedRows={dedupedRows}
```
Which is definitely confusing that we have 2 different sets of deduplicated rows.